types/three/three-core.d.ts

```diff
@@ -5301,6 +5301,8 @@ export class Mesh extends Object3D {
 
     geometry: Geometry | BufferGeometry;
     material: Material | Material[];
+    customDepthMaterial: Material | undefined
```

Comment:
I believe this two properties are part of `Object3D` and not `Mesh`. Could you please change it to the right class?